eZ/Publish/SPI/Persistence/User/Policy.php

```diff
@@ -31,13 +31,14 @@ class Policy extends ValueObject
     public $roleId;
 
     /**
-     * Status of the policy (legacy: "original_id").
+     * Only used when the role's status, current policy belongs to, is Role::STATUS_DRAFT.
+     * Original policy ID the draft was created from.
      *
      * @since 6.0
      *
-     * @var int One of Role::STATUS_DEFINED|Role::STATUS_DRAFT
+     * @var int
```

Comment:
This is also null when not a draft right? And should document what value will be on newly created draft.